Controllers/LoginScreen.swift

```diff
@@ -65,7 +68,7 @@ class LoginScreen: UsersDatabase, UITextFieldDelegate {
         self.present(alert, animated: true)
       }
     } else if token != nil && (authentificationService?.oauth2.hasUnexpiredAccessToken())! {
-      performSegue(withIdentifier: "SegueFromLogScreenToConnected", sender: self)
+      performSegue(withIdentifier: "ShowInterventionVC", sender: self)
```

Comment:
Segue identifiers should be written in camelCase, with a first letter lowercased